app/controllers/identify_controller.rb

```diff
@@ -64,9 +64,14 @@ def referrer
         value = URI(value)
         value.query = ''
         value.fragment = ''
-        # TODO: LG-183 - make sure referrer is whitelisted
       end
-      value
+      value if value && allowed_referrer?(value)
     end
   end
+
+  # :reek:UtilityFunction
+  def allowed_referrer?(uri)
+    allowed_host = Figaro.env.identity_idp_host
+    !allowed_host || uri.host == allowed_host
```

Comment:
If I'm reading this correctly, this means that if `Figaro.env.identity_idp_host` is not set (it doesn't appear to be required), then `allowed_referrer?` will return `true`, which is not what we want, right?

Reply:
Correct. It's to prevent an ordering dependency between deployment and configuration. Once we have things rolled out and working, I'll go back and mark configurations as required. Just a lot of spinning plates/balls in the air right now.